recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# Architecture-dependent flags
+if [ "$(uname)" = "Darwin" ]; then
+    LIB_FLAG='-dynamiclib'
+    EXT='.dylib'
+    deployFlag="-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13"
+else
+    LIB_FLAG='-shared'
+    EXT='.so'
+    deployFlag=""
+fi
+
+CMAKE_ARGS+=("-DCMAKE_PREFIX_PATH=${PREFIX}")
+
+# Build dependencies first
+
+# Multiview
+cd $SRC_DIR/MultiView
+mkdir -p build && cd build
+cmake ..                                          \
+    -DCMAKE_BUILD_TYPE=Release                    \
+    -DMULTIVIEW_DEPS_DIR=${PREFIX}                \
+    -DCMAKE_VERBOSE_MAKEFILE=ON                   \
+    -DCMAKE_CXX_FLAGS="-O3 -std=c++11 -Wno-error" \
+    -DCMAKE_C_FLAGS='-O3 -Wno-error'              \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX}              \
+    $deployFlag
+make -j${CPU_COUNT} install
```

Comment:
Is there a make check or ctest that we can run here?

Reply:
> Given your preference, I added one test for one tool, in meta.yml. I think it is not reasonable to expect conda to ensure the correctness of all the code I ship.

It is not my preference. It is expected from all compiled Software to be tested in the different architectures and build systems they are shipped. Something that passes and is correct in the CIs upstream may fail and be broken here.